### PR TITLE
Add release package playbook

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
Once these updates are in along side (https://github.com/theforeman/foreman-infra/pull/380#issuecomment-345049174) I think we can start building out the actual repository.

File: add_package.yml [hunk 1]
```diff
@@ -27,3 +27,9 @@
       shell: "gem2rpm -t {{ rubygem_spec_template }} {{ gem_name }}-{{ version }}.gem > {{ inventory_hostname }}.spec"
       args:
         chdir: "packages/{{ inventory_hostname }}"
+
+    - name: 'Create git branch'
+      command: "git checkout -b {{ gem_name }}-version"
```
Comment (evgeni) [inline add_package.yml]: was that intended to be `{{ gem_name }}-{{ version }}`?

Comment (ehelms) [reply add_package.yml]: It was! Good catch

File: cleanup_copr.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+- hosts: localhost
+  gather_facts: no
+  tasks:
+    - name: 'Get Copr repositories'
+      command: "copr-cli list @theforeman"
+      register: copr_repos
+
+    - set_fact:
+        scratch_repos: "{{ copr_repos.stdout.split('\n') | select('match', 'Name') | list | replace('Name: ', '') }}"
```
Comment (ekohl) [inline cleanup_copr.yml]: Can you use regexes or check that the line also starts with ```Name: ```? Right now it could match a description.

**Comment by ehelms** [comment]
Updated the clean up script and created myself a task for adding a README as a follow up

**Pull request merged**